### ISSUE TITLE
Expected warning for non-initialized relation in body

### ISSUE THREAD

**Issue by Laschoking** [knowsys/nemo#352]
I would expect a warning, if a relation like "nonInitRelation" neither is imported nor appears in the head of any rule.
`relation(?a) :- 
    nonInitRelation(?a).`

**Comment by matzemathics** [comment]
@mmarx, I don't think this must necessarily be solved by the parser, because such "global" properties usually will make the parser much more complicated. I think this might should be solved on the same level as type analysis, though I guess there is no rigid architecture in place yet for such checks.

**Comment by mmarx** [comment]
Yeah, probably needs a better tag. Any suggestions?

**Comment by matzemathics** [comment]
> Yeah, probably needs a better tag. Any suggestions?

something like "Analysis" / "Program Analysis" ?
